### PR TITLE
topic 0

### PR DESCRIPTION
**Pull request by faltamiranda**

### CODE REVIEW

File: 00-HTML-CSS-basics/trabajos/css/2.1.4/index.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link href="style.css" rel="stylesheet" type="text/css" >
+    <meta charset="utf-8">
+    <title>Style page</title>
+  </head>
+  <body  style="background-color:#4f5151">
+
+<div class="header">
```
Comment (afiorenza) [inline 00-HTML-CSS-basics/trabajos/css/2.1.4/index.html:10]: Check indentation

File: 00-HTML-CSS-basics/trabajos/css/2.1.4/index.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  <div class="secfooter">
+<h1>Section Footer</h1>
+  </div>
+
+</div>
+
+<div class="aside">
+<h1>aside</h1>
+</div>
+
```
Comment (afiorenza) [inline 00-HTML-CSS-basics/trabajos/css/2.1.4/index.html:38]: Avoid the use of breaklines

File: 00-HTML-CSS-basics/trabajos/html/trabajo4.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Example 4</title>
+  </head>
+  <body>
+    <form>
+    First name: <input type="text" name="fname" id="fname" value=""  required="" /><br>
```
Comment (afiorenza) [inline 00-HTML-CSS-basics/trabajos/html/trabajo4.html:9]: Try to use labels

**Review by afiorenza — CHANGES_REQUESTED** [review]

File: 00-HTML-CSS-basics/trabajos/css/2.1.4/index.html (hunk 1 above)
Comment (faltamiranda) [reply 00-HTML-CSS-basics/trabajos/css/2.1.4/index.html:10]: done

File: 00-HTML-CSS-basics/trabajos/css/2.1.4/index.html (hunk 2 above)
Comment (faltamiranda) [reply 00-HTML-CSS-basics/trabajos/css/2.1.4/index.html:38]: done

File: 00-HTML-CSS-basics/trabajos/html/trabajo4.html (hunk 3 above)
Comment (faltamiranda) [reply 00-HTML-CSS-basics/trabajos/html/trabajo4.html:9]: done

**Review by afiorenza — APPROVED** [review]

**Pull request merged**